src/Users.js

```diff
@@ -57,7 +62,8 @@ class Users extends React.Component {
         params: {
           query: makeQueryFunction(
             'cql.allRecords=1',
-            '(username="%{query.query}*" or personal.firstName="%{query.query}*" or personal.lastName="%{query.query}*" or personal.email="%{query.query}*" or barcode="%{query.query}*" or id="%{query.query}*" or externalSystemId="%{query.query}*")',
+            // TODO: Refactor/remove this after work on FOLIO-2066 and RMB-385 is done
+            (parsedQuery, props, localProps) => localProps.query.query.split(' ').map(query => compileQuery({ query })).join(' or '),
```

Comment:
I thought we wanted to join clauses with `AND` since the terms within each clause are `ORed`.

Reply:
@zburke I think you are right here! It will narrow our results. Let me test this first.